site/docs/patterns/app-header.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: App header
+layout: DetailPattern
+aliases:
+  - /salt/patterns/app-header
+data:
+  resources:
+    [
+      { href: "https://www.saltdesignsystem.com/salt/components/navigation-item/examples", label: "Navigation item" },
+    ]
+---
+
+The app header is a consistently available element that sits on top of every application interface. It contains a brand logo or mark and application-wide functionality, including navigation and utility actions. This pattern introduces guidelines to make the app header consistent and usable. 
+
+  <Diagram
+    src="/img/patterns/app-header/app-header.png"
+    alt="App header example"
+    border
+  />
+
+## When to use
+
+The app header serves as the primary element for users to orient themselves within the UI. It should be globally persistent, enabling easy navigation and access to utility functions.
+
+## How to build
+
+### Anatomy
+
+The app header has three functional areas:
+
+1. **Branding:** Displays a brand logo or mark followed by the application name in text form. This should always link to the product’s home page.
+2. **Navigation:** Serves as the container for the primary navigation. Its inclusion is optional if you have single-page applications or use a persistent side navigation.
+3. **Utilities:** Provides access to persistent application-wide utilities, such as external links, search, notifications, help, account details and settings.
+
+  <Diagram
+    src="/img/patterns/app-header/anatomy.png"
+    alt="Header functional areas—branding, navigation and utilities."
+    border
+  />
+
+### Desktop layout construction
+
+1. Use [navigation item](/salt/components/navigation-item) to build the horizontal page navigation, as explained in the [navigation pattern](./navigation).
```

Comment:
@mikearildbrown please may you review this section and the mobile layout section